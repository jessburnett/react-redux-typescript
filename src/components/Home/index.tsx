import * as React from 'react';

import ApiCallExample from 'containers/ApiCallExample/index';

export const Home = (): JSX.Element => (
  <div>
    <h1>HOME</h1>
    <img src="/assets/images/flower.jpg" />
    <h3>API call example:</h3>
    <ApiCallExample />
  </div>
);

export default Home;
