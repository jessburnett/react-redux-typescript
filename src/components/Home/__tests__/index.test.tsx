import * as React from 'react';
import { shallow } from 'enzyme';

import Home from '../';

describe('<Home/>', () => {

    it('renders correctly', () => {
        expect(shallow(<Home/>)).toMatchSnapshot();
    });

});
