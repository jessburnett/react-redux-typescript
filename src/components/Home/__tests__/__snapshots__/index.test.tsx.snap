exports[`<Home/> renders correctly 1`] = `
<div>
  <h1>
    HOME
  </h1>
  <img
    src="/assets/images/flower.jpg" />
  <h3>
    API call example:
  </h3>
  <Connect(ApiCallExample) />
</div>
`;
